### PR TITLE
Ddrage

### PR DESCRIPTION
**Pull request by HenningTimm**
* [x] I have read the guidelines above.
* [x] This PR adds a new recipe.

### CODE REVIEW

File: recipes/ddrage/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+    - randomize_fastq = ddrage.tools.randomize_fastq:main
+    - learn_qmodel = ddrage.tools.learn_qmodel:main
+
+  # If this is a new build for the same version, increment the build
+  # number. If you do not include this key, it defaults to 0.
+  # number: 1
+
+requirements:
+  build:
+    - python >=3.5
```
Comment (bgruening) [inline recipes/ddrage/meta.yaml]: Can you remove the pinning here. You already skip python2 builds in your build section.

File: recipes/ddrage/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    - setuptools
+    - numpy
+    - numba
+    - matplotlib
+    - scipy
+    - pyyaml
+
+  run:
+    - python >=3.5
```
Comment (bgruening) [inline recipes/ddrage/meta.yaml]: Same here.

File: recipes/ddrage/meta.yaml (hunk 1 above)
Comment (HenningTimm) [inline recipes/ddrage/meta.yaml]: I did, but I have code that will not work with Python 3.4 and lower. This crashed during the tests last time.

**Comment by HenningTimm** [comment]
Thanks for the help. I rebased the branch. I ran out of ideas yesterday.

**Comment by bgruening** [comment]
@HenningTimm I think we ave it. But this branch is still broken. To many changes in this commit. Can you fix this or simply create a new PR with the recipe?

**Comment by HenningTimm** [comment]
@bgruening Thank you for spending so much time on this. A created a new PR (https://github.com/bioconda/bioconda-recipes/pull/4210) with only the final version and will delete this PR.

**Pull request closed without merging**